src/app/features/auth/pages/login/login-page.component.ts

```diff
@@ -31,20 +31,22 @@ export class LoginPageComponent {
     private readonly _toastr: ToastrService
   ) {}
 
-  public async login(formData: LoginData): Promise<void> {
-    const [error] = await to(
-      firstValueFrom(
-        this._store.dispatch(new Login(formData, formData.remember))
+  public async login(formData: LoginCredentials): Promise<void> {
+    const [error] = await to<Login, HttpErrorResponse>(
+      firstValueFrom<Login>(
+        this._store
+          .dispatch(new Login(formData))
+          .pipe(catchError((e: HttpErrorResponse) => Promise.reject(e)))
       )
     )
 
     if (error) {
-      const loginError = (error as HttpErrorResponse).error as ResponseError
+      const loginError = error.error as ErrorResponse
 
-      this._toastr.error(loginError.message, loginError.error )
+      this._toastr.error(loginError.message, loginError.error)
 
       return
     }
-    void await this._router.navigate(['/'])
+    await this._router.navigate(['/'])
```

Comment:
void or return would be a better suit here, because you are not interested in the result and you do not want to wait for it. Nothing changes technically, just lexically better.

Reply:
cannot use void because the eslinter waits for 'await' for the async function call. 
if I use return then I also should change the return type of the function to Promise<boolean>, and all the return; replace with return false; 
In my opinion, I will make a function more complex just for lexical perfection.